packages/data-explorer-ui/src/components/Filter/components/SearchAllFilters/searchAllFilters.styles.ts

```diff
@@ -20,3 +20,9 @@ export const GroupHeading = styled.div`
 export const GroupDivider = styled(Divider)`
   margin: 8px 0;
 `;
+
+export const MatchHighlight = styled.mark`
+  background: ${({ theme }) => theme.palette.warning.light};
+  padding: 1px 0 2px 0;
```

Comment:
The unpadded highlight looked noticeably shorter than the mock, but I wasn't certain how best to fix that -- this is what I came up after experimenting to find a reasonable approximation of the mock (at least on my screen)

Reply:
@hunterckx the <span> and <mark> element do not behave like a block element; they are described as inline elements. To apply the background and for the element to behave like a block element we can apply `display: inline-block` to the styles.